ssn/integrated/ssn-dul.ttl

```diff
@@ -75,13 +75,21 @@ sosa:Sensor
 ## ProcedureExecutions
 
 sosa:usedProcedure
-  rdfs:subPropertyOf dul:satisfies ;
+  rdfs:subPropertyOf dul:hasParticipant ;
   rdfs:isDefinedBy sosa: .
 
 sosa:Observation
   rdfs:subClassOf dul:Event ;
   rdfs:isDefinedBy sosa: .
 
+ [ owl:intersectionOf  ( not(old-ssn:Observation)
```

Comment:
This is not syntactically correct, what was the intended meaning of these axioms ?